Bottom: 680c62e7ee0de8c626e4c5904c749b89409bf152
Top:    7b8b6c8a81e57805f13bc1f35008f77e8c14b980
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-13 01:48:59 +0100

hid/batch: #include "hid/common/draw_helpers.h" which was missing

Fixes a compiler warning about an implicitly defined function.


---

diff --git a/src/hid/batch/batch.c b/src/hid/batch/batch.c
index b528448..101dad9 100644
--- a/src/hid/batch/batch.c
+++ b/src/hid/batch/batch.c
@@ -18,6 +18,7 @@
 #include "../hidint.h"
 
 #include "hid/common/hidnogui.h"
+#include "hid/common/draw_helpers.h"
 #include "hid/common/actions.h"
 #include "hid/common/hidinit.h"
